### ISSUE TITLE
Document usage from code (non cli case)

### ISSUE THREAD

**Issue by wtrocki** [aerogear/graphback#237]

**Comment by wtrocki** [comment]
I think we will need to have general refactor of the architecture:

- graphback should be renamed to graphback-core package 
- graphback-cli should be renamed to graphback (but we should still publish cli for backwards compability
- we going to introduce other packages we worked on the side.

**Comment by ankitjena** [comment]
Yes we can rename them, but about usage from code currently we are generating a complex structure for resolvers output.

**Comment by wtrocki** [comment]
Yes.. We need to kinda see graphback-core where people can come in and write their own resolver outputs for their use cases. We are missing documentation for that as long as good api that will close all lose ends.
